platforms-serverless/serverless-framework-lambda/serverless.yml

```
@@ -15,3 +15,9 @@ functions:
     name: e2e-tests-serverless-lambda
     environment:
       SERVERLESS_LAMBDA_PG_URL: ${env:SERVERLESS_LAMBDA_PG_URL}
+
+# only include the Prisma binary required on AWS Lambda while packaging
+package:
+  patterns:
+    - '!node_modules/.prisma/client/query-engine-*'
+    - 'node_modules/.prisma/client/query-engine-rhel-*'
```

`query-engine-rhel-*` means that if the current binary target is `rhel-openssl-1.1.x` (1.1) for example you are developing on Linux then both 1.0 and 1.1 would be uploaded to Lambda.

I think Lambda only requires `rhel-openssl-1.0.x`?

---

True. Is that something we want to codify in these rules though? Seems like we have to, otherwise we could end up with 2 different binaries deployed again :/

---

```suggestion
    - 'node_modules/.prisma/client/query-engine-rhel-openssl-1.0.x'
```
I think it's better to be safe, this should do it